### DISCUSSION TITLE
HTML to text renderer

### DISCUSSION THREAD

**Discussion by dipeshSam** [flet-dev/flet#2063] [Ideas]
**Flet must have HTML_Text control as a container that can take HTML text as string and can render it supporting CSS and javascript.**

**Comment by Craftint** [comment]
Can we use other python libraries to handle this ? im looking for the same

**Reply by dipeshSam** [reply]
Then you need to use `Tkinter`. It provides a module named as HTMLLable. You can search how to use it.

**Comment by vindevoy** [comment]
Can't you do this via the webview control ?  Just write your html + css to disk and open it in webview ?

**Reply by dipeshSam** [reply]
It should be as this. But tii the date it only takes Website link, so.
